chat-sample is not Windows friendly

`npm install` fails:

```
PS C:\Users\alros\repos\Microsoft\vscode-extension-samples\chat-sample> npm install

> chat-sample@0.1.0 postinstall
> npm run setupTiktoken


> chat-sample@0.1.0 setupTiktoken
> cp node_modules/@vscode/prompt-tsx/dist/base/tokenizer/cl100k_base.tiktoken out/

'cp' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code 1
npm ERR! path C:\Users\alros\repos\Microsoft\vscode-extension-samples\chat-sample
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c npm run setupTiktoken

npm ERR! A complete log of this run can be found in: C:\Users\alros\AppData\Local\npm-cache\_logs\2024-05-30T13_14_26_042Z-debug-0.log
PS C:\Users\alros\repos\Microsoft\vscode-extension-samples\chat-sample>
```

--- a/chat-sample/package.json
+++ b/chat-sample/package.json
@@ -51,13 +51,16 @@
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "setupTiktoken": "cp node_modules/@vscode/prompt-tsx/dist/base/tokenizer/cl100k_base.tiktoken out/",
-        "postinstall": "npm run setupTiktoken"
+        "postinstall": "npm run setupTiktoken",
+        "setupTiktoken": "run-script-os",
+        "setupTiktoken:linux:darwin": "cp node_modules/@vscode/prompt-tsx/dist/base/tokenizer/cl100k_base.tiktoken out/",
+        "setupTiktoken:windows": "copy node_modules\\@vscode\\prompt-tsx\\dist\\base\\tokenizer\\cl100k_base.tiktoken out\\"
     },
     "devDependencies": {
         "@types/node": "^20.5.9",
         "@vscode/prompt-tsx": "^0.1.10-alpha",
         "eslint": "^7.22.0",
+        "run-script-os": "^1.1.6",
         "tslint": "^6.1.3",
         "typescript": "^4.0.3"
     }

